### ISSUE TITLE
misison video embed seems to pop out too far

### ISSUE THREAD

**Issue by tairea** [tairea/galaxy-maps-v2#59]
![Image](https://github.com/tairea/galaxy-maps-v2/assets/37652388/4a2e59f7-0496-4683-93c8-109ea15cb001)

**Comment by occhristian** [comment]
I can work on this.

**Comment by tairea** [comment]
That would be cool man. Thanks. Go for it. Push fix to a new branch, then create a pull request, then link pull request to this issue.
Also join our discord so we meet properly: https://discord.gg/XWZwmarXn

**Comment by occhristian** [comment]
I do not have push access to the repo.
I get a 403 error when I try to *git push *to create a new branch from my
local repo.

[image: image.png]

**Comment by tairea** [comment]
just added you to the project as a contributor 👍, try now

**Comment by occhristian** [comment]
Thank you.
Changes pushed. Pull request created, and linked to this issue.

Thanks for the opportunity to contribute.
There's still a lot of work to be done, as I can see.
I tried to reach out on Discord, but maybe discord is not my thing.

**Comment by tairea** [comment]
thanks man. will review now. 

next time see if you can link pull request to the same issue (#59) instead of creating another issue (#65). im still getting used to git as well 😅

no probs bout discord, you're mad early to the community, we the only people in there lol and i was sleeping so it probably looked broken lol

can i ask how you came across this project? im just starting out trying to build a community around the project

**Comment by occhristian** [comment]
Rather than sitting around doing nothing, I decided to look for open source
projects to contribute to.
I think your label “good first issue” made your project show up on
goodfirstissue.com or so.

I cloned the site and run it, and it was cool; so I chose it as my first
project to contribute to.

I’ve been to many of the galaxies, and it’s a really novel approach you
have employed. The work you have done so far is really astonishing.
